package.json

```diff
@@ -21,6 +21,7 @@
     "prepublishOnly": "yarn build:clean && yarn lint && yarn test",
     "setup": "yarn install && yarn allow-scripts",
     "test": "jest",
+    "posttest": "jest-it-up",
```

Comment:
note: `posttest` is not automatically run in Yarn v3, so this needs to be added if this template starts using Yarn v3.

Reply:
Yeah, we should proactively work out a solution here. Why use a `post*` hook here at all? In the `test` script can we say `jest && jest-it-up`?

Reply:
Hm, yeah, that's better.

Reply:
531110d